static/app/components/events/interfaces/crashContent/exception/stackTrace.spec.tsx

```diff
@@ -0,0 +1,187 @@
+
+import ExceptionStacktraceContent from 'sentry/components/events/interfaces/crashContent/exception/stackTrace';
+import {STACK_VIEW} from 'sentry/types/stacktrace';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+
+const frames = [
+  {
+    function: null,
+    colNo: null,
+    vars: {},
+    symbol: null,
+    module: '<unknown module>',
+    lineNo: null,
+    errors: null,
+    package: null,
+    absPath: 'https://sentry.io/hiventy/kraken-prod/issues/438681831/?referrer=slack#',
+    inApp: false,
+    instructionAddr: null,
+    filename: '/hiventy/kraken-prod/issues/438681831/',
+    platform: null,
+    context: [],
+    symbolAddr: null,
+    rawFunction: null,
+    trust: null,
+  },
+  {
+    absPath: 'foo/baz.c',
+    colNo: null,
+    context: [],
+    errors: null,
+    filename: 'foo/baz.c',
+    function: null,
+    inApp: false,
+    instructionAddr: null,
+    lineNo: 1,
+    module: null,
+    package: null,
+    platform: null,
+    rawFunction: null,
+    symbol: null,
+    symbolAddr: null,
+    trust: null,
+    vars: null,
+  },
+];
+
+const stacktrace: React.ComponentProps<typeof ExceptionStacktraceContent>['stacktrace'] =
+  {
+    framesOmitted: null,
+    hasSystemFrames: false,
+    registers: {},
+    frames,
+  };
+
+const props: React.ComponentProps<typeof ExceptionStacktraceContent> = {
+  platform: 'node',
+  expandFirstFrame: true,
+  newestFirst: true,
+  chainedException: false,
+  event: {
+    ...TestStubs.Event(),
+    entries: [],
+    crashFile: {
+      sha1: 'sha1',
+      name: 'name.dmp',
+      dateCreated: '2019-05-21T18:01:48.762Z',
+      headers: {'Content-Type': 'application/octet-stream'},
+      id: '12345',
+      size: 123456,
+      type: 'event.minidump',
+    },
+    culprit: '',
+    dateCreated: '2019-05-21T18:00:23Z',
+    'event.type': 'error',
+    eventID: '123456',
+    groupID: '1',
+    id: '98654',
+    location: 'main.js',
+    message: 'TestException',
+    platform: 'native',
+    projectID: '123',
+    tags: [{value: 'production', key: 'production'}],
+    title: 'TestException',
+  },
+  data: stacktrace,
+  stacktrace,
+  hasHierarchicalGrouping: false,
+  groupingCurrentLevel: undefined,
+  meta: undefined,
+};
+
+describe('ExceptionStacktraceContent', function () {
+  const organization = TestStubs.Organization();
+
+  it('default behaviour', function () {
+    const {container} = render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent {...props} />
+      </OrganizationContext.Provider>
+    );
+
+    expect(container).toSnapshot();
+  });
+
+  it('should return an emptyRender', function () {
+    render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent {...props} stacktrace={null} />
+      </OrganizationContext.Provider>
+    );
+
+    expect(screen.queryByText(frames[0].filename)).not.toBeInTheDocument();
```

Comment:
This is very brittle, for example if the component ever changes to display a different property than `filename`, this test will still pass. If you can assert that it renders emptyRender (maybe by adding data-testid if nothing is returned) it would make the test less easy pass

Reply:
yes, I agree.